test/model/test/model_basic/main.cpp

```diff
@@ -1223,9 +1222,8 @@ test_model_oldest_wt(void)
     wt_model_assert(table, uri, key1, 40);
     wt_model_assert(table, uri, key1, 50);
 
-    /* FIXME-WT-12412: Return an error if the provided timestamp is older than the current one. */
-    /* Test moving the oldest timestamp backwards - this should fail silently. */
-    /* wt_model_set_oldest_timestamp_both(10); */
+    /* Test moving the oldest timestamp backwards - this should fail. */
+    wt_model_set_oldest_timestamp_both(10);
```

Comment:
Is this call missing a `testutil_check(... == EINVAL)` wrapper?

Reply:
The definition of `wt_model_set_oldest_timestamp_both` includes an assertion that both WT and the model return the same return value.

Reply:
Got it - thanks.